drivers/timer/cortex_m_systick.c

```diff
@@ -202,7 +203,13 @@ void z_clock_set_timeout(s32_t ticks, bool idle)
 		delay =
 		 ((delay + CYC_PER_TICK - 1) / CYC_PER_TICK) * CYC_PER_TICK;
 		delay -= unannounced;
-		last_load = delay;
+		if (delay <= MAX_CYCLES) {
+			last_load = delay;
+		} else {
+			/* If delay contains a value that cannot be programmed,
+			 * leave the previous value of last_load into SysTic.
+			 */
+		}
```

Comment:
Swap the next two lines?  (i.e. put the blank line after the last closing brace)